src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,223 @@
 export const mappings = {
+  '_default_': {
+    'dynamic': 'false'
+  },
+  dashboard: {
```

Comment:
It feels like these mappings should be a part of each individual plugin's definition rather than crammed into the elasticsearch plugin, doesn't it? Feel free to ignore me if that's already the plan and this is just a stopgap.